public/app/features/alerting/partials/notification_edit.html

```diff
@@ -65,6 +65,14 @@ <h3 class="page-heading">Slack settings</h3>
       </div>
     </div>
 
+    <div class="gf-form-group" ng-if="ctrl.model.type === 'victorops'">
+      <h3 class="page-heading">Victorops settings</h3>
```

Comment:
I think for user-facing text, we should use VO's name correctly, VictorOps, capital V and O.